src/components/card/cardWrapper/CardWrapper.module.css

```diff
@@ -0,0 +1,26 @@
+:root {
+  --conduction-card-padding-inline-start: 18px;
+  --conduction-card-padding-inline-end: 18px;
+  --conduction-card-padding-block-start: 18px;
+  --conduction-card-padding-block-end: 18px;
+  --conduction-card-background-color: #ffffff;
+  --conduction-card-border-radius: 16px;
```

Comment:
I'm thinking that borders are quite common in cards, and having a bit more options on the border could be useful.

Two ways we could do that:

1. Create a token for the border itself, holding all three values (e.g. `1px solid black`;
2. Create three separate tokens for the values.